plugins/modules/win_certificate_info.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Ansible, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_certificate_info
+short_description: Get information on certificates from a Windows Certificate Store
+description:
+- Returns information about certificates in a Windows Certificate Store.
```

Comment:
We need to set the version when this module was added to the collection through the `version_added` key.

```suggestion
- Returns information about certificates in a Windows Certificate Store.
version_added: 2.6.0
```

Reply:
The reason you put 2.6.0 because it will be the upcoming version release, right?

Is there a place where we can follow the planned release dates if there is such

Reply:
The version is in the [galaxy.yml](https://github.com/ansible-collections/ansible.windows/blob/main/galaxy.yml) file and while sometimes it may have already been bumped it may still track the older release. I just compare it with the [tagged releases](https://github.com/ansible-collections/ansible.windows/releases) and see if I need to increment it or not. In this case `2.5.0` is the latest release and the `galaxy.yml` has not been updated yet. A minor version bump is needed for new features so the next release will be `2.6.0`.

As for a release schedule there hasn't been a hard rule of when to release. I usually aim for a release every 1-2 months but that is also dependent on whether there are changes to be made in a release.